packages/kanso-app-settings/build/compile.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 function get_doc(ddoc_url, settings, callback) {
     couchr.get(ddoc_url, function(err, resp){
         if (err && err.error === 'unauthorized') return auth(ddoc_url, settings, callback);
-        if (err && err.error === 'not_found') return callback(null, {}); // no design doc. First push.
+        if (err && err.response.statusCode === 404) return callback(null, {}); // no design doc. First push.
```

Comment:
@alxndrsn Yeah that looks sensible - maybe couch 2 is closing the connection quickly rather than buffering unnecessarily? No idea. We could patch couchr so that both of those conditions copy across the `reason` and `error` properties which would fix the issue. Or we could patch kanso to always look at the response (like you've done).

I'm not too worried about contributing to either of these projects as they're obviously EOLed and we are working on plans to move off kanso altogether.

This is fine for now. I'll leave it up to you whether you'd like to contribute to couchr or kanso-app-settings . Maybe just raise an issue so the next person knows what's up?